.claude/skills/run-ci-tests/bin/report.py

```
@@ -0,0 +1,199 @@
+    logs = [p for p in glob.glob(os.path.join(log_dir, "*.log")) if not os.path.basename(p).startswith("chain-")]
+    infos = [parse_log(p) for p in logs]
+    rank = lambda i: (ORDER.index(i["name"]) if i["name"] in ORDER else len(ORDER), i["name"])
+    infos.sort(key=rank)
+    L = [f"# {title}", "", f"Logs: `{os.path.abspath(log_dir)}/` (every claim below is grep-able there).", ""]
+    L += ["| Suite | Exit | Duration | Packages ok/FAIL | Tests PASS/FAIL/SKIP (top-level) | Log |", "|---|---|---|---|---|---|"]
+    overall_ok = True
+    for info in infos:
+        c = counts(info)
+        s = summary.get(info["name"], {})
+        ex = s.get("exit", info["exit"])
+        if ex is None: verdict = "⏳ no exit marker (still running or aborted)"; overall_ok = False
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)